UnitsNet.Benchmark/Initializations/UnitAbbreviationsCacheInitializationBenchmarks.cs

```diff
@@ -52,20 +52,20 @@ public string WithSpecificQuantityAndCustomMapping()
     public string DefaultWithoutLookup()
     {
         var cache = UnitAbbreviationsCache.CreateDefault();
-        return cache.GetAbbreviations(Mass.Info.BaseUnitInfo)[0];
+        return Mass.GetAbbreviation(Mass.BaseUnit);
     }
 
     [Benchmark]
     public string EmptyWithoutLookup()
     {
         var cache = new UnitAbbreviationsCache();
-        return cache.GetAbbreviations(Mass.Info.BaseUnitInfo)[0];
+        return Mass.GetAbbreviation(Mass.BaseUnit);
     }
 
     [Benchmark]
     public string WithSpecificQuantityWithoutLookup()
     {
         var cache = new UnitAbbreviationsCache([Mass.Info]);
-        return cache.GetAbbreviations(Mass.Info.BaseUnitInfo)[0];
+        return Mass.GetAbbreviation(Mass.BaseUnit);
```

Comment:
I'm not sure I understand what this is testing? `cache` is never used, so we test both constructing a new local `UnitAbbreviationsCache` instance + a lookup of the global default `UnitAbbreviationsCache`?

Reply:
Originally I was using this overload for testing the initialization of the `UnitAbbreviationsCache`, as it avoids the call to the `QuantityInfoLookup` but now that the `Mass.GetAbbreviation` call does it instead (using the internal overload)- I could switch it over and hopefully remove the method (see the TODO - we shouldn't accept a `UnitInfo` as an argument, as it might not be the same `UnitInfo` that was registered earlier).

Reply:
the overload (and it's one test) have been removed in https://github.com/angularsen/UnitsNet/pull/1548/commits/f9a2766b959334432c531c44afca207f37c155cc

Reply:
>cache is never used
Oh I see, this isn't doing anything anymore.. I'll just remove them.